Mathlib/Data/Set/Lattice.lean

```diff
@@ -2024,14 +2025,10 @@ theorem image2_iUnion_right (s : Set α) (t : ι → Set β) :
   simp only [← image_prod, prod_iUnion, image_iUnion]
 #align set.image2_Union_right Set.image2_iUnion_right
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
```

Comment:
These might be worth keeping as a reminder that `⋃ (i) (j), f i j` syntax is ugly and we still can't write the Lean 3 `⋃ i j, f i j`; or alternatively, we should just switch to `⋃ i, ⋃ j, f i j` below, if we think that's nicer.

Reply:
Reverted.